compiler/control/OMROptions.cpp

```diff
@@ -3681,6 +3659,28 @@ OMR::Options::jitPostProcess()
    if (_hotMaxStaticPICSlots < 0)
       _hotMaxStaticPICSlots = -_hotMaxStaticPICSlots * _maxStaticPICSlots;
 
+   if (self()->getOption(TR_EnableLargePages))
+      {
+      self()->setOption(TR_EnableLargeCodePages);
+      }
+
+#if defined(TR_HOST_64BIT)
```

Comment:
I realize this was in the original code as well, but shouldn't this be TR_TARGET_64BIT ?